docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Material-UI is now MUI!
+description: Starting today, we are evolving our brand identity. We are clarifying the difference between our company and our products.
+date: 2021-09-15T00:00:00.000Z
+authors: ['oliviertassinari', 'danilo-leal', 'mbrookes']
+card: true
+---
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/intro.png" alt="The new MUI logo" style="width: 100%; margin-bottom: 16px;" />
+
+Starting today we are evolving our brand identity to clarifying the difference between our company and our products.
+
+- Material-UI: the organization is now called [**MUI**](https://github.com/mui-org/).
+- Material-UI: the set of foundational MIT React components is now called [**MUI Core**](https://github.com/mui-org/material-ui).
+- Material-UI X: the set of advanced React components is now called [**MUI X**](https://github.com/mui-org/material-ui-x).
+
+Our previous name was no longer serving our areas of focus.
+We have grown our product offering.
+We needed a new identity to match our increased scope.
+
+## The origin
+
+Material-UI [started](https://www.text-em-all.com/blog/material-ui-the-internet-found-our-ui-project) as a React implementation of the Material Design guidelines in 2014.
+The goal was simple, to empower React developers to use Material Design.
+The community [loved it](https://news.ycombinator.com/item?id=8582439) and was craving for more, so we kept building.
+
+We made it easy for the community to contribute and started actively reviewing pull requests and guiding issues.
+Over 2,200 developers have cared enough about our mission to contribute, so far.
+Things started to really take off with [the release of v1](/blog/material-ui-v1-is-out/) in 2018.
+The frontend community was actively migrating to React, but didn't have the bandwidth to rebuild legacy interfaces from scratch,
+so before we knew it, we had grown into the most popular React UI component library.
+The components were heavily used and customized to build customer-facing applications, internal tools, and mobile-hybrid apps.
+
+## A larger focus
+
+In our last survey, the number of developers that commented about improving the Material Design implementation was down by [60%](/blog/2020-developer-survey-results/#comparison-with-last-year) compared to the year before.
```

Comment:
@oliviertassinari are you going to update the links to point back to https://mui.com/* after you import the blogpost on medium?

Reply:
No need, it's done automatically